lib/digital_public_works/posts.ex

```diff
@@ -51,10 +57,11 @@ defmodule DigitalPublicWorks.Posts do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_post(%Project{} = project, attrs \\ %{}) do
+  def create_post(%{"project" => project, "user" => user} = attrs \\ %{}) do
```

Comment:
@nickgartmann what is a good way to handle passing multiple associated structs to a context method? Both my current solution, and the previous solution of passing positional arguments feels a bit wrong

Reply:
```
attrs = Map.merge(attrs, %{ "project_id" => project.id, "user_id" => user.id})
```